dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WC5JakartaServletTest.java

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.fat.wc.tests;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.ws.fat.util.LoggingTest;
+import com.ibm.ws.fat.util.SharedServer;
+import com.ibm.ws.fat.wc.WCApplicationHelper;
+
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+
+/**
+ * Test Servlet 5 native jakarta.servlet (i.e not transformer)
+ * Should only run in EE9.
+ * The target app is servlet 5.0 version with new namespace.
+ * Request to the simple snoop servlet 50 version which test several jakarta.servlet API (along with the new web-app web.xml namespace)
+ */
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+@SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
+public class WC5JakartaServletTest extends LoggingTest {
+
+    private static final Logger LOG = Logger.getLogger(WC5JakartaServletTest.class.getName());
+    private static final String APP_NAME = "Servlet5TestApp";
+
+    @ClassRule
+    public static SharedServer SHARED_SERVER = new SharedServer("servlet50_wcServer");
+
+    /*
+     * (non-Javadoc)
+     *
+     * @see com.ibm.ws.fat.util.LoggingTest#getSharedServer()
+     */
+    @Override
+    protected SharedServer getSharedServer() {
+        return SHARED_SERVER;
+    }
+
+    @BeforeClass
+    public static void before() throws Exception {
+        LOG.info("Setup : add " + APP_NAME + " war to the server if not already present.");
+
+        WCApplicationHelper.addWarToServerDropins(SHARED_SERVER.getLibertyServer(), APP_NAME + ".war", true, "servlet5snoop.war.servlets");
+
+        ArrayList<String> expectedErrors = new ArrayList<String>();
+        expectedErrors.add("CWWWC0401E:.*");
+        SHARED_SERVER.getLibertyServer().addIgnoredErrors(expectedErrors);
```

Comment:
What error is this and why are we expecting it? Can we add some comments so we know in the future?